Update View.yml

lokeshchdhry opened the pull request:
1. Removed 2.0 reference in `auto` property in `Size & Position` section.
2. Removed `\n` from `backgroundDisabledImage` property.
3. Removed `\n` from `backgroundFocusedImage` property.
4. Removed `\n` from `backgroundImage` property.
5. Removed `\n` from `backgroundSelectedImage` property.

lokeshchdhry committed:
Update View.yml

1. Removed 2.0 reference in `auto` property in `Size & Position` section.
2. Removed `\n` from `backgroundDisabledImage` property.
3. Removed `\n` from `backgroundFocusedImage` property.
4. Removed `\n` from `backgroundImage` property.
5. Removed `\n` from `backgroundSelectedImage` property.

build commented:
<!--
  1 failure:  Tests have failed...
  1 warning:  There is no linke...
  2 messages
  1 markdown notices
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Tests have failed, see below for more information.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  There is no linked JIRA ticket in the PR body. Please include the URL of the relevant JIRA ticket. If you need to, you may file a ticket on <a href="https://jira.appcelerator.org/secure/CreateIssue!default.jspa">JIRA</a>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :thumbsup: Hey!, You deleted more code than you added. That's awesome!
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :floppy_disk: <a href="https://jenkins.appcelerator.org/job/titanium-sdk/job/titanium_mobile/job/PR-10348/1/artifact/dist/mobilesdk-7.5.0.v20180925132605-osx.zip">Here's the generated SDK zipfile</a>.
  </td>
    </tr>
  </tbody>
</table>

### Tests: 

| Classname | Name | Time | Error |
| --- | --- | --- | --- |
| ios.Titanium.UI | .AUTOFILL_TYPE_ONE_TIME_CODE | 0 | file:///Users/build/Library/Developer/CoreSimulator/Devices/F12B9A21-DCD |
| ios.Titanium.UI | .AUTOFILL_TYPE_NEW_PASSWORD | 0 | file:///Users/build/Library/Developer/CoreSimulator/Devices/F12B9A21-DCD |
| ios.Titanium.App.iOS.UserActivity | #deleteAllSavedUserActivities() | 0 | file:///Users/build/Library/Develo |
| ios.Titanium.App.iOS.UserActivity | #deleteSavedUserActivitiesForPersistentIdentifiers() | 0 | file:///Users/build/Library/Develo |

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

jawa9000 committed:
Updated View.yml

While reviewing the changes to this yaml file for 3rd quarter Doc Day 2018, I discovered that the `default` parameter was being heavily abused. The changes in this PR mostly moves errant content that should be listed in a `description` parameter.

jawa9000 reviewed (changes requested):
While the changes look good, I discovered several issues with the use of the `default` parameter. I created https://github.com/appcelerator/titanium_mobile/pull/10348/files#diff-0cc7dd4ed3fbabcc1ea4d811f65da9dc in response.

sgtcoolguy committed:
Merge branch 'master' into lokeshchdhry-patch-2

sgtcoolguy committed:
Merge branch 'master' into lokeshchdhry-patch-2
